src/App.js

```diff
@@ -1,11 +1,57 @@
 import "./App.css";
+import Header from "./components/Header.js";
+import ToDoList from "./components/ToDoList.js";
+import { useState } from "react";
+
+// const toDoArray = [
+//   { name: "shoppingItem", status: "Pending" },
+//   { name: "running", status: "Done" },
+// ];
+
+export default function App() {
+  const [toDo, setToDo] = useState([]);
+
+  function handleAddToDo(toDoItem) {
+    const newToDo = [
+      ...toDo,
+      {
+        name: toDoItem,
+        status: false,
+      },
+    ];
+    setToDo(newToDo);
+  }
+
+  function handleDeleteToDo(name) {
+    const removedToDo = toDo.filter((toDo) => toDo.name !== name);
+    setToDo(removedToDo);
+  }
+
+  function togglePendingButton(buttonContent) {
+    /* iteriere über alle toDos, schaue nach, ob es ein todo gibt, weclhes den gleichen namen hat, wie "buttonContent" (toDo.name === buttonContent),
+    dann ändere für exakt dieses todo den status */
+    const newButtonContent = toDo.map((status) => {
```

Comment:
I'd rename `status` to `toDo` because you are mapping over your `toDos` array.

You can check the Array method cheatsheet we gave you to get a refresher, or the documentation we gave you about `map` in the second JavaScript week